src/main/HomeScreen.js

```diff
@@ -42,31 +44,39 @@ class HomeScreen extends PureComponent<Props> {
     return (
       <View style={styles.wrapper}>
         <View style={styles.iconList}>
-          <NavButton
-            name="globe"
-            onPress={() => {
-              dispatch(doNarrow(HOME_NARROW));
-            }}
-          />
-          <NavButton
-            name="star"
-            onPress={() => {
-              dispatch(doNarrow(STARRED_NARROW));
-            }}
-          />
-          <NavButtonGeneral
-            onPress={() => {
-              dispatch(doNarrow(MENTIONED_NARROW));
-            }}
-          >
-            <IconUnreadMentions color={BRAND_COLOR} />
-          </NavButtonGeneral>
-          <NavButton
-            name="search"
-            onPress={() => {
-              NavigationService.dispatch(navigateToSearch());
-            }}
-          />
+          <View style={{ width: windowWidth * 0.25 }}>
```

Comment:
Let's see if we can use [flexbox](https://reactnative.dev/docs/flexbox) styling here.

One reason is that I expect it'll be more concise, but another is more subtle: [`Dimensions.get`](https://reactnative.dev/docs/dimensions) doesn't hook the React component into changes in the dimensions and cause the `render` method to run again. It just tells you what the dimensions are when `render` happens to run. The dimensions will at least change when the device is rotated, and probably at other times.